sdk/eventhub/azure-eventhub/azure/eventhub/aio/_error_async.py

```diff
@@ -70,7 +70,7 @@
             elif isinstance(exception, errors.AMQPConnectionError):
                 await closable._close_connection_async()
             elif isinstance(exception, compat.TimeoutException):
-                pass  # Timeout doesn't need to recreate link or connection to retry
+                await closable._close_connection_async()
```

Comment:
This no longer needs to be a separate check:
```
elif isinstance(exception, (errors.AMQPConnectionError, compat.TimeoutException)):
    await closable._close_connection_async()
else:
    ...
```
Same comment applies to `exceptions.py` line 204

Reply:
Combined them